### DISCUSSION TITLE
Customise message: "This PR was configured for branch automerge, however this is not possible so it has been raised as a PR instead"

### DISCUSSION THREAD

**Discussion by ps-jay** [renovatebot/renovate#18401] [Suggest an Idea]
Hey folks,

I've looked through the documentation, but I cannot see anything that lets me customise the message _"This PR was configured for branch automerge, however this is not possible so it has been raised as a PR instead"_

Ideally, I'd like to add a link to my company's documentation that tells folks how to authorise Renovate to skip the requirement to create PR's

Have I missed it in the documentation?
Should we work on a improvement PR instead?

Regards,
  Phil

**Comment by rarkins** [comment]
You could refactor the code to add it to https://docs.renovatebot.com/configuration-options/#userstrings which would then allow you to customize it

**Comment by ps-jay** [comment]
Thanks @rarkins !
